test/library/draft/arrow/Makefile

```diff
@@ -6,16 +6,15 @@ CHPL = chpl
 CHPLFLAGS = -g
 
 GLIB_INCLUDES = -I /usr/local/include/glib-2.0 -I /usr/local/lib/glib-2.0/include
-GLIB_LINKS = -larrow-glib -lglib-2.0 -lgobject-2.0
+GLIB_LINKS = -larrow-glib -lparquet-glib -lglib-2.0 -lgobject-2.0
```

Comment:
Did this additional link needed to be added with all the parquet stuff or is it because of the new headers? Like it seemed like we were able to use parquet without this before, so I am just curious.

Reply:
This is because I had a statement in the ParquetDecl.chpl file that went as follows:
```chapel
require "-lparqet-glib";
```
Which should not have been there in the first place.
Deleting that file made me realize that this link is not present in the makefile or the `.compopts` file and so I added it.

Reply:
Cool, makes sense. Looks good